README.md

```diff
@@ -22,8 +32,12 @@ $ npx openapi-transformer-toolkit
 
 The package includes CLI commands for easier usage. Use the `openapi-transformer-toolkit` executable followed by the command and required options:
 
-- `openapi-transformer-toolkit oas2json -i < input > -o < output >` - Generate JSON schemas from an OpenAPI file
-- `openapi-transformer-toolkit json2ts -i < input > -o < output > [-c < config >]` - Generate TypeScript types from JSON schemas
+- Generate JSON schemas from OpenAPI:
+  - `openapi-transformer-toolkit oas2json -i < input > -o < output > [-m]`
+- Generate TS types from OpenAPI:
+  - `openapi-transformer-toolkit oas2ts -i < input > -o < output > [-c < config > -m]`
+- Generate TS types from JSON schemas:
```

Comment:
do we need to keep this? I mean when would you want to generate ts types from json schemas? Is this tool generic and can generate types from any json schema or does it expect the schemas to be in a specific format (e.g. that generated by the oas2json?) in the first case it may be worth keeping, in the second I don't see much value in keeping this command

Reply:
The command `json2ts` doesn't expect a special format from `oas2json`. I just re-tested it with an example taken from: https://json-schema.org/understanding-json-schema/reference/object.html

I would keep it in the package, it adds flexibility and control in my opinion.

Reply:
fair enough